java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java

```diff
@@ -30,8 +30,12 @@
   private final String DEVICES_KEY = "devices";
   private final String TYPE_KEY = "type";
   private final String DATA_KEY = "data";
+  private final String METADATA_KEY = "metadata";
+  private final String REQUESTURL_KEY = "requesturl";
+  private final String SITEURL_KEY = "siteurl";
```

Comment:
"siteurl" is not currently used by PWS, nor has it been previously suggested going forward.

Would be worthwhile to share names as much as possible -- let's discuss tomorrow.